Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -158,13 +153,17 @@ namespace BabylonNative
             }
         }
 
+        void newEngine()
```

Comment:
```suggestion
        void NewEngine()
```

But it's still kind of a weird name. Maybe `Initialize()`?

Reply:
I agree, the name is weird. Initialize is too generic and I'd like to send the message that this function is called when a new engine is created. `EngineInitializationPending` ?

Reply:
I don't think it's too generic. It's called from the static Initialize function. The code happens to be setting a flag for engine, but it could potentially do other things.